.github/workflows/build.yml

```diff
@@ -19,19 +19,14 @@ jobs:
         ruby:
           - 3.1.2
           - 3.0.4
-          - 2.7.6
-          - 2.6.10
-          - 2.5.8
+          - truffleruby-head
```

Comment:
I don't think we want to test against head, we don't even do that with MRI. Maybe we could add some stable versions instead

Reply:
Agree.